portal-network.md

```diff
@@ -55,8 +61,13 @@ TODO
 
 ### Transaction Sending: Cooperative Transaction Gossip
 
-TODO
+The goal of the transaction gossip network is to make sure all new transactions are validated and broadcast to the miners so that they can be included in a block.
+
+Stateless clients should be able to declare where they stand on the spectrum of "how many transactions do I want to process out of the whole mempool", based on the amount of resources they have and their role in the network, and the network should let clients filter and select destination nodes based on which transactions they are interested in.
+
+Stateless transaction validation involves checking accounts' balances and nonces, so transactions should be broadcast with those proofs included.
 
+Transaction broadcasting involves pushing new transactions to the "center" of the network, where the miners are.  Note how this is different from the needs of the state network, where pieces of the state need to be "pushed out" from the full nodes to the light ones who will store it so they can participate in achieving or strenghening consensus.  This means that the "lightest" clients would not have to know about all (or any) transactions, they just need to make sure their own transactions are validated and sent to the miners as quickly as possible.
```

Comment:
I think references to the "center" of the network get too close to implementation details.  While we're pretty sure that the network will be structured this way, it's not actually part of the requirements.  The more generic requirements are that:

1. Nodes can "tune" how much of the mempool they are required to process
2. Nodes can perform nonce and balance validation checks on transactions without a local copy of the state
3. Nodes can "tune" their network participation such that they receive the full mempool of transactions.

I think (2) is not yet captured in your description.